### ISSUE TITLE
use a incorrect hidden layer in softmax layer

### ISSUE THREAD

**Issue by cml54** [mulhod/tensorflow_tutorials#1]
In `utils.py`, I found a incorrect part: https://github.com/mulhod/tensorflow_tutorials/blob/master/tutorial_notebooks/src/utils.py#L188

`hidden1` was used in softmax layer. But, it should be the last hidden layer. It means that hidden3 needs to be used if the network has three hidden layers.

**Comment by mulhod** [comment]
Thanks for catching that. I've since fixed it.
